src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+  align-items: center;
+  width: 96px;
+  height: 100%;
+
+  &__star {
+    height: 100%;
+    width: 20%;
```

Comment:
set width using `px`

Reply:
Why?

Reply:
Because if something changes you markup will be broken. 
Example: if you use 10 stars instead of 5 ones.
That's why it's better to set width directly to a each star instead of their parent wrapper